build/builder.sh

```diff
@@ -32,6 +32,7 @@ if [ "${1-}" = "version" ]; then
   exit 0
 fi
 
+GOPATH=${GOPATH-$HOME/go}
```

Comment:
This could also be `GOPATH=$(go env GOPATH)` to ask go for the right value without having to duplicate it.